Source/santa-driver/SantaDecisionManager.h

```diff
@@ -290,13 +328,18 @@ class SantaDecisionManager : public OSObject {
   uint32_t failed_log_queue_requests_;
 
   int32_t listener_invocations_;
+  int32_t pid_monitor_thread_count_ = 0;
 
   pid_t client_pid_;
 
   kauth_listener_t vnode_listener_;
   kauth_listener_t fileop_listener_;
 
   struct timespec ts_;
+
+  // When set to true, Santa will check all ancestors of a process to determine
+  // if it is a compiler.
```

Comment:
There doesn't appear to be a way to make this true - is it just for testing or are we going to add a driver method to enable it?

Reply:
Eventually there should be a way to change it along with other parameters that are currently defined as constants.  Added a TODO for this and moved this to live with the other constants.